src/dust/dust_cuda.cu

```diff
@@ -1,12 +1,23 @@
+/*!
+ * \file dust_cuda.cu
+ * \author Helena Richie (helenarichie@gmail.com)
+ * \brief Contains code that updates the dust density scalar field. The dust_kernel function determines the rate of
+ * change of dust density, which is controlled by the sputtering timescale. The sputtering timescale is from the
+ * McKinnon et al. (2017) model of dust sputtering, which depends on the cell's gas density and temperature.
+ */
+
 #ifdef CUDA
   #ifdef DUST
 
+    // STL includes
     #include <stdio.h>
 
+    // External includes
     #include <cstdio>
     #include <fstream>
     #include <vector>
 
+    // Local includes
```

Comment:
This isn't strictly part of the PR. I think that the includes for `dust_cuda.h` should include the full path: `../dust/dust_cuda.h` to be consistent with all the other includes in the code. It also enables easy automated changes if a file gets moved